packages/ethers/src/client.ts

```diff
@@ -565,7 +544,7 @@ export class Web3Modal extends Web3ModalScaffold {
     localStorage.removeItem(EthersConstantsUtil.WALLET_ID)
     EthersStoreUtil.reset()
 
-    if (providerType === 'injected' || providerType === 'eip6963') {
+    if (providerType === 'eip6963') {
```

Comment:
Just double checking that the provider type isn't persisted in `localStorage` to avoid people getting in a broken state.

Reply:
If a users latest connection was with the injected connector, it will lose the connection. They need to reconnect (with EIP6963). They won't get into a broken state where they can't disconnect.